src/pages/iou/IOUModal.js

```diff
@@ -155,6 +156,20 @@ class IOUModal extends Component {
         PersonalDetails.fetchCurrencyPreferences();
     }
 
+    /**
+     * Decides our animation type based on whether we're increasing or decreasing
+     * our step index.
+     * @returns {String}
+    */
+    getAnimation() {
+        if (this.state.previousStepIndex < this.state.currentStepIndex) {
+            return 'slideInRight';
+        }
+        if (this.state.previousStepIndex > this.state.currentStepIndex) {
+            return 'slideInLeft';
+        }
+    }
```

Comment:
For example, IOUModal should not be responsible for determining the animation.

Reply:
Makes sense! Just to make sure I'm understanding: you think it would be better for the `IOUModal` to only pass the direction we're going to our Animated component instead of the animation type directly?

Reply:
Yep, exactly. Then we won't need to define the `getAnimation` function in every other Modal that has steps.